helpers/kernel_symbols.go

```diff
@@ -0,0 +1,102 @@
+package helpers
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+)
+
+/*
+ * This package gives the ability to hold all the known kernel symbols.
```

Comment:
This is in the helpers package (which is good, don't need another one), but it would be more accurate to say "The helpers in this file gives the ability..."